samples/bot-conversation-sso-quickstart/js/package.json

```diff
@@ -1,21 +1,36 @@
 {
-    "name": "teams-conversation-bot",
+    "name": "teams-bot",
     "version": "1.0.0",
-    "msteams": {
-        "teamsAppId": "6ec02800-7667-472e-a73c-b6b3e8340d06"
-    },
-    "description": "Microsoft Teams conversation bot quickstart",
+    "description": "Bot Builder v4 Bot Teams sample",
     "author": "Microsoft",
     "license": "MIT",
     "main": "index.js",
     "scripts": {
         "start": "node ./index.js",
-        "watch": "nodemon ./index.js"
+        "watch": "nodemon ./index.js",
+        "lint": "eslint .",
+        "test": "echo \"Error: no test specified\" && exit 1"
+    },
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/Microsoft/BotBuilder-Samples.git"
     },
     "dependencies": {
-        "botbuilder": "~4.7.0",
-        "dotenv": "^8.0.0",
-        "express": "~4.17.1",
-        "html-entities": "^1.3.1"
+        "@microsoft/microsoft-graph-client": "^2.2.1",
+        "botbuilder": "~4.11.0",
+        "botbuilder-dialogs": "~4.11.0",
+        "crypto": "^1.0.1",
+        "dotenv": "^8.2.0",
+        "isomorphic-fetch": "^3.0.0",
+        "restify": "~8.5.1"
+    },
+    "devDependencies": {
+        "eslint": "^7.0.0",
+        "eslint-config-standard": "^14.1.1",
```

Comment:
Not sure if we need add .eslint.rc and related lib in the dev dependency, maybe user don't use eslint for linter

Reply:
We don't need it, it was generated my my vscode's lint plugin. It was not exist in other sample projects, so I delete it.